src/main/java/bio/overture/rollcall/config/ElasticsearchConfig.java

```diff
@@ -19,31 +19,37 @@
 package bio.overture.rollcall.config;
 
 import lombok.SneakyThrows;
+import lombok.extern.slf4j.Slf4j;
 import lombok.val;
 import org.apache.http.HttpHost;
 import org.apache.http.auth.AuthScope;
 import org.apache.http.auth.UsernamePasswordCredentials;
+import org.apache.http.conn.ssl.TrustSelfSignedStrategy;
 import org.apache.http.impl.client.BasicCredentialsProvider;
+import org.apache.http.ssl.SSLContextBuilder;
 import org.elasticsearch.client.RestClient;
 import org.elasticsearch.client.RestHighLevelClient;
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 
-import java.net.URL;
+import java.security.KeyManagementException;
+import java.security.KeyStoreException;
+import java.security.NoSuchAlgorithmException;
 
 @Configuration
+@Slf4j
 public class ElasticsearchConfig {
 
-  @Value("${elasticsearch.host}")
-  private String host;
+  @Value("${elasticsearch.node}")
+  private String node;
 
-  @Value("${elasticsearch.port}")
-  private int port;
-
-  @Value("${elasticsearch.authEnabled}")
+  @Value("${elasticsearch.authEnabled:false}")
   private boolean authEnabled;
 
+  @Value("${elasticsearch.trustSelfSignedCert:false}")
```

Comment:
the defaults are set in the application.yaml, it's confusing to set a property in different places